### PR TITLE
add support status for COS regarding CWS distributions

### PR DESCRIPTION
**Pull request by paulcacheux**
### What does this PR do? What is the motivation?

This PR adds a new entry in the supported distributions table for CWS regarding the "preview support" for container optimized OS.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/threats/supported_linux_distributions.md [hunk 1]
```diff
@@ -4,16 +4,17 @@ title: CSM Threats Supported Linux Distributions
...
+|---------------------------------------------------------------|-------------------------|
+| Ubuntu LTS                                                    | 18.04, 20.04, 22.04     |
+| Debian                                                        | 10 or later             |
+| Amazon Linux 2                                                | Kernels 4.14 and higher |
+| Amazon Linux 2023                                             | All versions            |
+| SUSE Linux Enterprise Server                                  | 12 and 15               |
+| Red Hat Enterprise Linux                                      | 7, 8, and 9             |
+| Oracle Linux                                                  | 7, 8, and 9             |
+| CentOS                                                        | 7                       |
+| Google Container Optimized OS (default on GKE) (PREVIEW)      | 93 and higher           |
```
Comment (maycmlee) [inline content/en/security/threats/supported_linux_distributions.md:17]:
```suggestion
| Google Container Optimized OS (default on GKE) (Preview)      | 93 and higher           |
```

**Review by maycmlee — COMMENTED** [review]
Just a small edit!

File: content/en/security/threats/supported_linux_distributions.md (hunk 1 above)
Comment (paulcacheux) [reply content/en/security/threats/supported_linux_distributions.md:17]: I was reading https://datadoghq.atlassian.net/wiki/spaces/PM/pages/3310617880/Updated+Product+Lifecycle+Datadog and understood that we are supposed to use all uppercase, do you confirm this is not the case ?

**Review by maycmlee — APPROVED** [review]
@paulcacheux is this ready to merge? If so, feel free to comment `/merge` to merge.

**Comment by paulcacheux** [comment]
I'm waiting for the PM approval, but yeah I will /merge once I get it. Thanks !

**Pull request merged**